openedx.yaml

```diff
@@ -0,0 +1,8 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification
+
+nick: drfx
+oeps: {}
+openedx-release: {ref: release}
```

Comment:
I don't think we want `openedx-release` set for repos representing packages that are installed in Open edX services.  It could be confusing if we tagged the latest master at the time of the release as `open-release/hawthorn`, but the version actually used in Hawthorn was an earlier commit released to PyPI.